mysql-test/suite/innodb/t/insert_into_empty.test

```diff
@@ -26,9 +26,19 @@ DROP TEMPORARY TABLE t;
 SET @save_ahi = @@global.innodb_adaptive_hash_index;
 SET GLOBAL innodb_adaptive_hash_index = 1;
 CREATE TABLE t1(f1 INT NOT NULL)ENGINE=InnoDB;
+
+SET @old_bulk_op=
+(SELECT variable_value FROM information_schema.global_status
+WHERE variable_name = 'innodb_bulk_operations');
+
 BEGIN;
 INSERT INTO t1 SELECT * FROM seq_1_to_65536;
 ROLLBACK;
+
+SELECT variable_value-@old_bulk_op bulk_operations
+FROM information_schema.global_status
+WHERE variable_name = 'innodb_bulk_operations';
```

Comment:
I was a bit uncertain about this, but based on other tests, it appears that `-` is defined on string parameters, by converting the strings to integers first. (The type of `information_schema.global_status.variable_value` is string, not integer.)